tms/readme/CONFIGURE.md

```diff
@@ -0,0 +1,47 @@
+TMS module can be used with advanced configuration that enable extra features complementing the base module.
+
+### Units of Measure
```

Comment:
```suggestion
## Units of Measure
```

While conducting some tests, I needed to make this change to avoid errors in the generation of the readme.

Reply:
all ### to ##